motoman_driver/MotoPlus/MotionServer.c

```diff
@@ -1751,6 +1755,13 @@ void Ros_MotionServer_IncMoveLoopStart(Controller* controller) //<-- IP_CLK prio
 			{
 				if(ret == -3)
 					printf("mpExRcsIncrementMove returned: %d (ctrl_grp = %d)\r\n", ret, moveData.ctrl_grp);
+#if (YRC1000||YRC1000u)
+				else if (ret == -1 || ret == -19)
+				{
+					printf("mpExRcsIncrementMove returned: %d (TFS Active)\r\n", ret, moveData.ctrl_grp);

```

Comment:
Is TFS another name for PFL?

Reply:
No, that's for the source control in Visual Studio.  I got my accronyms mixted up.  I'll make the correction.

Reply:
I've pushed a commit which fixes this.

Reply:
@gavanderhoorn 
Thanks for the update.  I had made the changes locally but I was waiting for confirmation on the -19 return value which is actually undocumented. 
-3 is E_EXRCS_CTRL_GRP
-1 is E_EXRCS_IMOV_UNREADY
-19 is I believe is PFL related but I don't know exactly in which context it occurs.  Normally I see -1.  So, I'm trying to get the confirmation from Japan and an update on the header files that should define it.

Reply:
> I had made the changes locally but I was waiting for confirmation

yes, I should probably have done a PR against your branch.

Apologies.